### PR TITLE
Create a protobuf schema.

### LINKED ISSUE #18: Define spec in protobuf
Using protobuf as a source of truth gives several benefits

- type safety
- generation of OpenAPI spec/JSON

### PR DESCRIPTION
**Pull request by MarkLodato**
Closes #18.

### CODE REVIEW

File: envelope.proto [hunk 1]
```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.intoto;
+
+// An authenticated message of arbitrary type.
+message Envelope {
+  // Message to be signed. (In JSON, this is encoded as base64.)
+  bytes payload = 1;
```
Comment (mnm678) [inline envelope.proto:8]: Do these fields need to be marked as required/optional?

Comment (MarkLodato) [reply envelope.proto:9]: Good point. Done.

I also added an explanation of le64(), len(), and utf8().

**Comment by MarkLodato** [comment]
@colek42 Could you please make sure that this indeed meets your needs for #18? Thanks!

**Review by adityasaky — APPROVED** [review]

**Review by mnm678 — APPROVED** [review]

**Comment by colek42** [comment]
LGTM, I think @mswift wanted to take a look at this too

**Pull request merged**